content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,234 @@
+
+</InfoBox>
+
+
+Palette provides an installer in the form of a docker container that is temporarily deployed on your laptop, workstation or jump-box. You can run the installer on any system that has a docker daemon installed and connectivity to Palette and the MAAS identity endpoint.
+
+
+# Prerequisites
+
+- A linux computer with a docker daemon installed and a connection to Palette and the MAAS endpoint. The installer must be invoked on a linux system.
+
+- One IP address for a Kubernetes control-plane.
+
+- One IP address for a single-node gateway or three IP addresses for a three-node gateway.
+
+- Sufficient IPs for application workload services, such as Load Balancer services.
+
+<WarningBox>
+
+By default, the MAAS Kubernetes pack uses 192.168.0.0/16. Ensure that the Pod CIDR range for any clusters you deploy after setting up the PCG do not overlap with the network used by the bare metal machines that MAAS manages.
+
+</WarningBox>
+
+- Minimum capacity requirements as follows:
+
+    - One node (no HA): 4 vCPU, 8 GiB Memory, 60 GiB Storage.
+    - Three nodes (HA): 6 vCPU, 12 GiB Memory, 90 GiB Storage.<br /><br />
+
+- An active MAAS API key in the format  ``[consumer_key], [key], and [secret] tokens: API key = '[consumer_key]:[key]:[secret]'``.
+
+- The DNS server that the installer will use must be able to resolve the public internet names of the machines that MAAS manages because the installer must connect to the machines.  
+
+    The installer must connect to the MAAS machines. This is commonly done by ensuring the DNS server delegates the MAAS domain to the MAAS controlplane. To connect, the installer uses the FQDN ``machine-hostname.maas``. A common way to enable this is to ensure that the DNS server delegates the MAAS domain to the MAAS controlplane.
+
+
+# Install the Gateway
+
+`video: title: "maas-pcg-creation": /pcg-creation-video/maas.mp4`
+
+Follow these steps to install the PCG. 
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+2. From the **User Menu**, select Cluster Mode and choose **Tenant Admin**.
+
+3. Navigate to the **Main** menu and select **Tenant Settings > Private Cloud Gateway** below the table.
+
+4. Select **MAAS**. Private Gateway installation instructions are displayed.
+
+5. Copy the pairing code displayed on the instructions page. You will input this code when you run the installer.
+
+6. Copy the following commands to your terminal to invoke the installer.
+    <br />
+
+    ```bash
+    docker run -it --rm \
+    --net=host \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v /tmp:/opt/spectrocloud \
+    gcr.io/spectro-images-public/release/spectro-installer:1.0.11 \
+    -o true
+    ```
+
+7. When prompted, enter information listed in each of the following tables. The installer will generate the gateway configuration file. 
+    <br />
+
+    <WarningBox>
+
+    When specifying the MAAS endpoint, be sure not to add a trailing / .
```

Comment:
```suggestion
    When specifying the MAAS endpoint, do not add a trailing `/` as it can result in DNS resolution errors.
```
Explaining the why 😃

Reply:
thanks for that! Added it.